src/Writer/XLSX/AutoFilter.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenSpout\Writer\XLSX;
+
+use OpenSpout\Common\Exception\InvalidArgumentException;
+
+final class AutoFilter
+{
+    /** @var string */
+    private $range = '';
+
+    public function __construct(string $range = '')
+    {
+        $this->range = $range;
+    }
+
+    /**
+     * Get AutoFilter Range.
+     */
+    public function getRange(): string
+    {
+        return $this->range;
+    }
+
+    /**
+     * Set AutoFilter Cell Range.
+     */
+    public function setRange(string $range): self
+    {
+        if ('' === $range) {
+            $this->range = '';
+
+            return $this;
+        }
+
+        $preg_match_all = preg_match_all('/([A-Z]+)(\d+):([A-Z]+)(\d+)/', $range, $matches);
```

Comment:
To keep consistency with the library, we can just copy-paste what's already done in `MergeCell`, and call `CellHelper::getColumnLettersFromColumnIndex` in the `FileSystemHelper` only.